website/app/components/doc/tokens-list/item.hbs

```diff
@@ -3,7 +3,8 @@
   SPDX-License-Identifier: MPL-2.0
 }}
 
-<li class="doc-tokens-list__item">
+{{! Add role="listitem" because the class sets display: contents; }}
```

Comment:
```suggestion
{{! role="listitem" is needed here because the class sets `display: contents` and some browsers and assistive technologies will ignore the implied role }}
```